btree/btree.c

```diff
@@ -1130,9 +1141,12 @@ struct level {
 	/** nd for newly allocated node at the level. **/
 	struct nd *l_alloc;
 
-	/** flag for indicating if node needs to get freed. **/
+	/**
+	 * This is the flag for indicating if node needs to get freed. This flag
+	 * will get set during delete operation and it will get used by
```

Comment:
s/This flag will get set during delete operation and it will get used by P_FREENODE phase for determine if node needs to get freed/Currently this flag is set in delete operation and is used by P_FREENODE phase for determine if node needs to be freed/

Reply:
done